ChatApp/ViewController.swift

```diff
@@ -1,19 +1,138 @@
 //
 //  ViewController.swift
-//  ChatApp
+//  chatApp.test
 //
-//  Created by Ana Dzebniauri on 20.04.23.
+//  Created by Ana Dzebniauri on 14.04.23.
 //
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
from the name ViewController it is not clear for what this given ViewController is for. Calling it for instance ChatViewController or something like this will make it little bit more clear